tools/deb/deb_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Test for istio debian. Will run in a docker image where the .deb has been installed.
+
+
+exit 0
```

Comment:
Various new files were added without a license.  I'm not clear on what kind of files in istio should have a license but it seemed worth a mention.  This file is missing a newline at the end.

Reply:
Fixed